tests/integration/test_integration.py

```diff
@@ -745,21 +746,29 @@ def test_downgrade(encoding, monkeypatch):
     log.error.assert_not_called()
 
 
-@allencodings
-def test_span_tags(encoding, monkeypatch):
-    monkeypatch.setenv("DD_TRACE_API_VERSION", encoding)
+def test_span_tags(run_python_code_in_subprocess):
+    code = """
+import mock
+from ddtrace import tracer as t
+
+with mock.patch("ddtrace.internal.writer.writer.log") as log:
```

Comment:
we don't need to mock this anymore do we?

can we just check that stdout and stderr are empty from the subprocess?